public/app/features/explore/spec/helper/setup.tsx

```diff
@@ -295,7 +294,7 @@ export function makeDatasourceSetup({
             // eslint-disable-next-line @grafana/no-untranslated-strings
             <div>
               <input
-                aria-label={t('explore.make-datasource-setup.aria-label-query', 'query')}
+                aria-label="query"
```

Comment:
can get rid of the comment just above this as well then 👍

Reply:
Oh yeah, oops, thanks